base-lite/ub/ub.go

```diff
@@ -0,0 +1,313 @@
+package main
```

Comment:
i don't know Go, so you need to get someone more familiar with this language to review this script and the tests. I would suggest getting someone on the kafka team since they would also be familiar with cub/dub a little.